exercises/robot/reflected_inertia.ipynb

```diff
@@ -0,0 +1,741 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "mcSz0hOVtBd5"
+   },
+   "source": [
+    "## Problem Description\n",
+    "You will notice that in the first few weeks of the lecture, we don't think a lot about the full second-order dynamics of the arm (a more complicated version of $F=ma$), although we will come to revisit this later. This problem set helps to understand one of the reasons why we can abstract many manipulation problems kinematically (purely in terms of joint angles and end-effector positions), as opposed to worrying about dynamics (joint torques and inertias). Even before going to the multi-link case, let's think about how we should control the position of a single pendulum. \n",
+    "\n",
+    "In this problem you will investigate the roll of reflected inertia in position control of a simple pendulum. At the end of the notebook, you will be able to check your work in a simulation environment. \n",
+    "\n",
+    "**These are the main steps of the exercise:**\n",
+    "1. Write down the state-space dynamics function for the simple pendulum with motor and gearbox.\n",
+    "2. Set up a PID Controller to do position control for the simple pendulum.\n",
+    "3. Qualitatively answer the difference in behavior between direct-drive and gearbox cases.\n",
+    "\n",
+    "\n",
+    "**Your job for this problem set**\n",
+    "Will be to do step 1 and 3 of the main steps.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "zGGyX76oBXGN"
+   },
+   "source": [
+    "## The Simple Pendulum\n",
+    "\n",
+    "<img src=\"https://raw.githubusercontent.com/RussTedrake/manipulation/master/figures/exercises/simple_pendulum.png\" width=\"400\">\n",
+    "\n",
+    "The [simple pendulum](http://underactuated.csail.mit.edu/pend.html) is perhaps the simplest abstraction of a 'manipulator', with the following equation for dynamics:\n",
+    "\n",
+    "$$ ml^2 \\ddot{q} = -mgl\\sin(q) + \\tau$$\n",
+    "\n",
+    "Let's first write down the dynamics of a pendulum in a first-order form, \n",
+    "$$\\dot{x}=f(x,u)$$\n",
+    "\n",
+    "where $x=[q,\\dot{q}]^T$ is now our state vector, and our input $u=\\tau$.\n",
+    "\n",
+    "$$\\frac{d}{dt}\\begin{bmatrix} q \\\\ \\dot{q} \\end{bmatrix}=\\begin{bmatrix} \\dot{q} \\\\ (-mgl\\sin(q) + \\tau) / ml^2 \\end{bmatrix}$$\n",
+    "\n",
+    "The following function returns $\\dot{x}$ given $x,u$, and the parameters. \n",
+    "\n",
+    "NOTE: For a sine implementation, we will be using `pydrake`'s `sin` for this problem set instead of the numpy implementation. For convenience, this can be called via `sin()` "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "6IRgS5ftCc71"
+   },
+   "outputs": [],
+   "source": [
+    "def pendulum_dynamics(x, u, p):\n",
```

Comment:
i was hoping that you would change this, but i guess it's ok that you haven't.

Reply:
I left a comment in the old doc, but I was wondering what you meant by the Context comment - is there a function signature to better incorporate your comment?